CMakeLists.txt

```diff
@@ -88,3 +88,10 @@ set(CPACK_ARCHIVE_COMPONENT_INSTALL ON)
 set(CPACK_COMPONENTS_ALL bin)
 
 include(CPack)
+
+# Create a custom build target called "docker" that will invoke DockerBuild.sh and create the NiFi-MiNiFi-CPP Docker image
+add_custom_target(
+    docker
+    COMMAND ${CMAKE_SOURCE_DIR}/docker/DockerBuild.sh 1000 1000 0.2.0 nifi-minifi-cpp-0.2.0-bin.tar.gz ${CMAKE_SOURCE_DIR}
```

Comment:
Would be good to to make use of the defined project version variables to avoid having to update this.  A bit more verbose, but would automatically be taken care of on release/version incrementing.